### ISSUE TITLE
Activation script results in error 

### ISSUE THREAD

**Issue by BastianZim** [conda-forge/libxml2-feedstock#89]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

I'm seeing the following error when running` mamba update --all`: `/opt/homebrew/Caskroom/mambaforge/base/etc/conda/activate.d/libxml2_activate.sh:export:12: not valid in this context: XML_CATALOG_FILES+`

### Installed packages

```shell
# packages in environment at /opt/homebrew/Caskroom/mambaforge/base:
#
# Name                    Version                   Build  Channel
beautifulsoup4            4.11.2             pyha770c72_0    conda-forge
black                     23.1.0          py311h267d04e_0    conda-forge
blinker                   1.5                pyhd8ed1ab_0    conda-forge
boolean.py                3.7                        py_0    conda-forge
brotlipy                  0.7.0           py311he2be06e_1005    conda-forge
bzip2                     1.0.8                h3422bc3_4    conda-forge
c-ares                    1.18.1               h3422bc3_0    conda-forge
ca-certificates           2022.12.7            h4653dfc_0    conda-forge
cctools                   973.0.1             hcbb26d4_11    conda-forge
cctools_osx-arm64         973.0.1             hef52d2f_11    conda-forge
certifi                   2022.12.7          pyhd8ed1ab_0    conda-forge
cffi                      1.15.1          py311hae827db_3    conda-forge
chardet                   5.1.0           py311h267d04e_0    conda-forge
charset-normalizer        2.1.1              pyhd8ed1ab_0    conda-forge
click                     8.1.3           unix_pyhd8ed1ab_2    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
conda                     23.1.0          py311h267d04e_0    conda-forge
conda-build               3.23.3          py311h267d04e_0    conda-forge
conda-forge-pinning       2023.03.06.17.29.54      hd8ed1ab_0    conda-forge
conda-package-handling    2.0.2              pyh38be061_0    conda-forge
conda-package-streaming   0.7.0              pyhd8ed1ab_1    conda-forge
conda-smithy              3.23.0             pyhd8ed1ab_0    conda-forge
cryptography              39.0.2          py311h507f6e9_0    conda-forge
curl                      7.88.1               h9049daf_0    conda-forge
dataclasses               0.8                pyhc8e2a94_3    conda-forge
deprecated                1.2.13             pyh6c4a22f_0    conda-forge
expat                     2.5.0                hb7217d7_0    conda-forge
filelock                  3.9.0              pyhd8ed1ab_0    conda-forge
fmt                       9.1.0                hffc8910_0    conda-forge
gettext                   0.21.1               h0186832_0    conda-forge
git                       2.39.2          pl5321hd6a5f80_0    conda-forge
gitdb                     4.0.10             pyhd8ed1ab_0    conda-forge
gitpython                 3.1.31             pyhd8ed1ab_0    conda-forge
glob2                     0.7                        py_0    conda-forge
gmp                       6.2.1                h9f76cd9_0    conda-forge
icu                       70.1                 h6b3803e_0    conda-forge
idna                      3.4                pyhd8ed1ab_0    conda-forge
isodate                   0.6.1              pyhd8ed1ab_0    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
krb5                      1.20.1               h69eda48_0    conda-forge
ld64                      609                 h619f069_11    conda-forge
ld64_osx-arm64            609                 h7167370_11    conda-forge
libarchive                3.6.2                h83f22c9_0    conda-forge
libcurl                   7.88.1               h9049daf_0    conda-forge
libcxx                    15.0.7               h75e25f2_0    conda-forge
libedit                   3.1.20191231         hc8eb9b7_2    conda-forge
libev                     4.33                 h642e427_1    conda-forge
libffi                    3.4.2                h3422bc3_5    conda-forge
libiconv                  1.17                 he4db4b2_0    conda-forge
liblief                   0.12.3               hb7217d7_0    conda-forge
libllvm13                 13.0.1               hfd59cb2_2    conda-forge
libllvm14                 14.0.6               hf6e71e7_1    conda-forge
libmamba                  1.3.1                h7d1d596_2    conda-forge
libmambapy                1.3.1           py311hf90d2e5_2    conda-forge
libnghttp2                1.52.0               hae82a92_0    conda-forge
libsodium                 1.0.18               h27ca646_1    conda-forge
libsolv                   0.7.23               hb5ab8b9_0    conda-forge
libsqlite                 3.40.0               h76d750c_0    conda-forge
libssh2                   1.10.0               h7a5bd25_3    conda-forge
libxml2                   2.10.3               h87b0503_2    conda-forge
libzlib                   1.2.13               h03a7124_4    conda-forge
license-expression        1.2                        py_0    conda-forge
lz4-c                     1.9.4                hb7217d7_0    conda-forge
lzo                       2.10              h642e427_1000    conda-forge
mamba                     1.3.1           py311hb045da1_2    conda-forge
markupsafe                2.1.2           py311he2be06e_0    conda-forge
msrest                    0.6.21             pyh44b312d_0    conda-forge
mypy_extensions           1.0.0              pyha770c72_0    conda-forge
ncurses                   6.3                  h07bb92c_1    conda-forge
oauthlib                  3.2.2              pyhd8ed1ab_0    conda-forge
openssl                   3.0.8                h03a7124_0    conda-forge
packaging                 23.0               pyhd8ed1ab_0    conda-forge
patch                     2.7.6             h27ca646_1002    conda-forge
pathspec                  0.11.0             pyhd8ed1ab_0    conda-forge
pcre2                     10.40                hb34f9b4_0    conda-forge
perl                      5.32.1          2_h9b22ae9_perl5    conda-forge
pip                       23.0.1             pyhd8ed1ab_0    conda-forge
pkginfo                   1.9.6              pyhd8ed1ab_0    conda-forge
platformdirs              3.1.0              pyhd8ed1ab_0    conda-forge
pluggy                    1.0.0              pyhd8ed1ab_5    conda-forge
psutil                    5.9.4           py311he2be06e_0    conda-forge
py-lief                   0.12.3          py311ha397e9f_0    conda-forge
pybind11-abi              4                    hd8ed1ab_3    conda-forge
pycosat                   0.6.4           py311he2be06e_1    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pycryptodome              3.16.0          py311hd9c7c45_0    conda-forge
pygithub                  1.58.0             pyh1a96a4e_0    conda-forge
pyjwt                     2.6.0              pyhd8ed1ab_0    conda-forge
pynacl                    1.5.0           py311he2be06e_2    conda-forge
pyopenssl                 23.0.0             pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1              pyha2e5f31_6    conda-forge
python                    3.11.0          h3ba56d0_1_cpython    conda-forge
python-libarchive-c       4.0             py311h267d04e_2    conda-forge
python_abi                3.11                    3_cp311    conda-forge
pytz                      2022.7.1           pyhd8ed1ab_0    conda-forge
pyyaml                    6.0             py311he2be06e_5    conda-forge
readline                  8.1.2                h46ed386_0    conda-forge
reproc                    14.2.4               h1a8c8d9_0    conda-forge
reproc-cpp                14.2.4               hb7217d7_0    conda-forge
requests                  2.28.2             pyhd8ed1ab_0    conda-forge
requests-oauthlib         1.3.1              pyhd8ed1ab_0    conda-forge
ripgrep                   13.0.0               h65448a5_2    conda-forge
ruamel.yaml               0.17.21         py311he2be06e_2    conda-forge
ruamel.yaml.clib          0.2.7           py311he2be06e_1    conda-forge
ruamel_yaml               0.15.80         py311he2be06e_1008    conda-forge
scrypt                    0.8.18          py311hf6ed75f_4    conda-forge
setuptools                67.5.1             pyhd8ed1ab_0    conda-forge
shyaml                    0.6.2              pyhd3deb0d_0    conda-forge
sigtool                   0.1.3                h44b9a77_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
smmap                     3.0.5              pyh44b312d_0    conda-forge
soupsieve                 2.3.2.post1        pyhd8ed1ab_0    conda-forge
sqlite                    3.40.0               h2229b38_0    conda-forge
tapi                      1100.0.11            he4954df_0    conda-forge
tk                        8.6.12               he1e0b03_0    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
toolz                     0.12.0             pyhd8ed1ab_0    conda-forge
tqdm                      4.65.0             pyhd8ed1ab_1    conda-forge
typed-ast                 1.5.4           py311he2be06e_1    conda-forge
typing-extensions         4.4.0                hd8ed1ab_0    conda-forge
typing_extensions         4.4.0              pyha770c72_0    conda-forge
tzdata                    2022g                h191b570_0    conda-forge
urllib3                   1.26.14            pyhd8ed1ab_0    conda-forge
vsts-python-api           0.1.25             pyhd8ed1ab_1    conda-forge
wheel                     0.38.4             pyhd8ed1ab_0    conda-forge
wrapt                     1.15.0          py311he2be06e_0    conda-forge
xz                        5.2.9                h1a8c8d9_0    conda-forge
yaml                      0.2.5                h3422bc3_2    conda-forge
yaml-cpp                  0.7.0                hb7217d7_2    conda-forge
zlib                      1.2.13               h03a7124_4    conda-forge
zstandard                 0.19.0          py311hdcbfb07_1    conda-forge
zstd                      1.5.2                hf913c23_6    conda-forge
```

### Environment info

```shell
active environment : base
    active env location : /opt/homebrew/Caskroom/mambaforge/base
            shell level : 1
       user config file : /Users/bastianzimmermann/.condarc
 populated config files : /opt/homebrew/Caskroom/mambaforge/base/.condarc
                          /Users/bastianzimmermann/.condarc
          conda version : 23.1.0
    conda-build version : 3.23.3
         python version : 3.11.0.final.0
       virtual packages : __archspec=1=arm64
                          __osx=13.2.1=0
                          __unix=0=0
       base environment : /opt/homebrew/Caskroom/mambaforge/base  (writable)
      conda av data dir : /opt/homebrew/Caskroom/mambaforge/base/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/osx-arm64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /opt/homebrew/Caskroom/mambaforge/base/pkgs
                          /Users/bastianzimmermann/.conda/pkgs
       envs directories : /opt/homebrew/Caskroom/mambaforge/base/envs
                          /Users/bastianzimmermann/.conda/envs
               platform : osx-arm64
             user-agent : conda/23.1.0 requests/2.28.2 CPython/3.11.0 Darwin/22.3.0 OSX/13.2.1
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```

**Comment by ocefpaf** [comment]
xref.: https://github.com/conda-forge/admin-requests/pull/687

**Comment by BastianZim** [comment]
Thanks!

**Comment by BastianZim** [comment]
I've just updated this after https://github.com/conda-forge/admin-requests/pull/687 was merged and am still seeing this. Should be through the CDN by now, no?

**Comment by ocefpaf** [comment]
Ha! Updating won't remove the "newer" one installed in lieu for an older package. This is a problem that I've been thinking about for a while but we need a way to inform conda-update that a package was removed.

**Comment by BastianZim** [comment]
Ohh I was not aware of that, interesting. Maybe reinstalling it would help? Although that seems like a question for conda/mamba.

**Comment by jackjchen** [comment]
Temp fix/workaround for zsh is to do the assignment then export in the activate script.
https://github.com/conda-forge/libxml2-feedstock/blob/dd02d8c1fbf2a7181104677aacd954b94b6423ec/recipe/activate.sh#L12

**Comment by hattne** [comment]
Should be fixed as of #91.
